apps/modernization-ui/src/utils/orNull.ts

```diff
@@ -2,4 +2,14 @@ function orNull<T>(value: T | null | undefined): T | null {
     return (value && value) || null;
 }
 
-export { orNull };
+/*
+ * checks if an object has any defined properties
+ */
+function objectOrUndefined(object: any) {
+    if (object === undefined) {
+        return undefined;
+    }
+    return Object.values(object).find((e) => e !== undefined) ? object : undefined;
```

Comment:
`some` has the same signature as `find` but returns a boolean.  If this is only for objects can you make the type of `object` be `object`?